src/main/resources/application-local.yml

```diff
@@ -35,4 +38,8 @@ aws:
             group: /COMP_CCH_zqxmzQ6iEkTRw
```

Comment:
Not your change - but can you try and pull the `group`, `team`, and `user` values out of this group.

Reply:
Done.

Reply:
Removed.